sdk/identity/identity/test/manual/package.json

```diff
@@ -5,13 +5,14 @@
   "main": "dist/bundle.js",
   "scripts": {
     "build": "webpack --config webpack.config.js",
-    "start": "webpack-dev-server"
+    "start": "webpack-dev-server",
+    "format": "prettier --write \"src/**/*.tsx\""
   },
   "author": "Microsoft Corporation",
   "license": "MIT",
   "dependencies": {
-    "@azure/identity": "1.2.0",
-    "@azure/keyvault-keys": "4.1.0",
+    "@azure/identity": "../../azure-identity-1.2.3.tgz",
```

Comment:
This assumes the presence of the package.
Can you add a step in the readme to `rush build -t identity && rushx pack` before doing `npm install`?

Reply:
I will point to 1.2.4-beta.1 instead! How does that sound?

Reply:
It would still require users to do `rush build -t identity && rushx pack`.

Reply:
why if 1.2.4-beta.1 is going to be released by that point? 🤔

Reply:
That's fine then. I thought you meant `1.2.4-beta.1.tgz`.